### DISCUSSION TITLE
Input from before OMP is started is eated up again

### DISCUSSION THREAD

**Discussion by arjan-s** [JanDeDobbeleer/oh-my-posh#5628] [Troubleshoot] [answered]
Yesterday I updated oh-my-posh from 23.4.0 to 23.12.0, and this issue started happened again. The bot at the bottom of the existing issue said to make a discussion instead of a new issue, so here we are.
The solutions from https://github.com/JanDeDobbeleer/oh-my-posh/issues/5072 (adding `disable_cursor_positioning = true`) and https://github.com/JanDeDobbeleer/oh-my-posh/issues/3446 (removing `newline`) both don't help.

**Comment by JanDeDobbeleer** [comment, marked as answer]
@arjan-s see [here](https://github.com/JanDeDobbeleer/oh-my-posh/issues/5524#issuecomment-2348425708)

**Reply by arjan-s** [reply]
Oh thank you! I didn't find that issue before when searching for it. Yes, this solves it for me too!
I don't need or want upgrade notices, so this is perfect for me.
